src/components/StorageView.jsx

```diff
@@ -24,7 +25,7 @@ class StorageView extends Component {
     const diskUsage = Number.isInteger(storageData.usage)
       ? (storageData.usage / (1000 * 1000 * 1000)).toFixed(2)
       : storageData.usage
-    const percent = (diskUsage / diskQuota) * 100
+    const percent = computeDiskPercent(storageData.usage, storageData.quota)
```

Comment:
Wonder if we shouldn't have some like: 

const { percentUsage, humanDiskUsage, humanDiskQuota } = computeDiskUsage()

Reply:
That sounds good to me, implementing

Reply:
Done ✅

Reply:
You should have created a commit for this one ;)